fc/fullcontact.go

```diff
@@ -394,6 +398,127 @@ func (fcClient *fullContactClient) EmailVerification(email string) chan *APIResp
 	return ch
 }
 
+/* Permission
+FullContact Permission API - PermissionCreate, takes an PermissionRequest and returns a channel of type APIResponse.
+Request is converted to JSON and sends a Asynchronous request */
+func (fcClient *fullContactClient) PermissionCreate(permissionRequest *PermissionRequest) chan *APIResponse {
+	ch := make(chan *APIResponse)
+	if permissionRequest == nil {
+		go sendToChannel(ch, nil, "", NewFullContactError("Permission Request can't be nil"))
+		return ch
+	}
+	err := validateForPermissionCreate(permissionRequest)
+	if err != nil {
+		go sendToChannel(ch, nil, "", err)
+		return ch
+	}
+
+	reqBytes, err := json.Marshal(permissionRequest)
+	if err != nil {
+		go sendToChannel(ch, nil, "", err)
+		return ch
+	}
+	// Send Asynchronous Request in Goroutine
+	go fcClient.do(permissionCreateUrl, reqBytes, ch)
+	return ch
+}
+
+/* FullContact Permission API - PermissionDelete, takes an PermissionRequest and returns a channel of type APIResponse.
+Request is converted to JSON and sends a Asynchronous request */
+func (fcClient *fullContactClient) PermissionDelete(permissionRequest *MultifieldRequest) chan *APIResponse {
+	ch := make(chan *APIResponse)
+	if permissionRequest == nil {
+		go sendToChannel(ch, nil, "", NewFullContactError("Permission Request can't be nil"))
```

Comment:
This message can use MultifieldRequest instead of Permission Request.

Reply:
changed

Reply:
Error message is not updated